### PR TITLE
New modules for the HP Vertica Analytics Database

### PR DESCRIPTION
**Pull request by dareko**
The following new modules for the HP Vertica Analytics Database (http://www.vertica.com) are based on postrgresql_db and postrgresql_user modules:
- vertica_facts
- vertica_schema
- vertica_user
- vertica_role
- vertica_configuration

If you decide to add to Ansible, please also consider adding to the HP Haven Marketplace (https://haven.hpwsportal.com/catalog.html#/Category/%7B%22categoryId%22%3A10073%7D/Show).

### CODE REVIEW

**Comment by e0d** [comment]
@bcoca Any idea when this is feature will be reviewed/merged?  It's something we'd like to use.

**Comment by bcoca** [comment]
we are in the middle of releasing 1.9, once that is out we will start reviewing/merging new features again.

**Comment by bcoca** [comment]
@dareko, could you squash the commits?

**Comment by dareko** [comment]
I've just added the vertica_role and vertica_configuration modules and made some updates to the code.

@bcoca, please provide an instruction on how to properly sqash all the commits? It would be good to add it to this page as well: http://docs.ansible.com/developing_modules.html

**Comment by e0d** [comment]
@dareko I think all that @bcoca is asking for is:

git rebase -i onto the latest ansible:devel with the appropriate pick and squash options

Details here:  http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by dareko** [comment]
@bcoca, I've squashed all the commits.

File: database/vertica/vertica_configuration.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+- name: updating load_balance_policy
+  vertica_configuration: name=failovertostandbyafter value='8 hours'
+"""
+
+try:
+    import pyodbc
+except ImportError:
+    pyodbc_found = False
+else:
+    pyodbc_found = True
```
Comment (bcoca) [inline database/vertica/vertica_configuration.py:87]: you can just put the True after the import, let the except set false, don't need to change now

**Comment by bcoca** [comment]
no need to change this now, but modules can use put shared code and docs in module_utils

**Comment by dareko** [comment]
@bcoca, thank you for merging.
I'm making updates based on your comments and will add them later.
As for shared code and docs in module_utils, do you mean extending ansible/lib/ansible/module_utils/database.py, adding ansible/lib/ansible/module_utils/database_vertica.py or something else?

**Comment by bcoca** [comment]
adding a vertica.py or databas_vertica.py works for me

**Pull request merged**